### ISSUE TITLE
Sources not shown in RAG Demo

### ISSUE THREAD

**Issue by jwratten** [databricks-demos/dbdemos#129]
Hi, 

A screenshot from the RAG Demo notbeook, seen below, shows "Sources", but I do not see this when I actually run the demo.

<img width="550" alt="image" src="https://github.com/databricks-demos/dbdemos/assets/166432073/ed36d4fd-60f7-4975-9c6c-ed1414b6e69b">

Also, I was wondering if you had any suggestions for, in the case of a pdf or word document, to list as the source both the page number and the document name or something to this effect which would better point the user to which part of the document was actually used?

Thanks!

**Comment by QuentinAmbard** [comment]
I wonder if it's not an issue with serialization I remember we had that but I thought it was fixed.
Are you writing your langchain in a file and loading the file? Also you need to make sure you call the mlflow.models.set_model(model=chain) function
Can yo uitry with the chain in the 02-Simple-app folder?

**Comment by jwratten** [comment]
I did now try with the 02-simple-app and same results.

**Comment by jwratten** [comment]
In the process of trying the 02-simple-app I also ran into an issue with the Lakehouse app as shown below.
![image](https://github.com/databricks-demos/dbdemos/assets/166432073/8de08252-4da0-44b4-99e9-84856b2495f4)

**Comment by QuentinAmbard** [comment]
yes for the lakehouse apps it's still in preview and I think not enabled. I'll add a better message to clarify it

**Comment by jwratten** [comment]
Ok, thanks. Did you happen to have any update on the original issue I mentioned?

**Comment by QuentinAmbard** [comment]
Hey @jwratten , We're having a look internally to see what could be happening, stay tuned!

**Comment by jwratten** [comment]
Hi, we have now started getting this error... Don't think we were getting it two weeks ago, any thoughts?
![image](https://github.com/user-attachments/assets/c29b2220-e0b8-40cf-a5b1-959176e85087)
